source/agora/consensus/Ledger.d

```diff
@@ -1058,7 +1098,7 @@ public class Ledger
 
     ***************************************************************************/
 
-    public Hash getRandomSeed (in Height height, in uint[] missing_validators)
+    private Hash[] getRandomSeed (in Height height, in uint[] missing_validators)
```

Comment:
```suggestion
    private Hash[] getPreImages (in Height height, in uint[] missing_validators)
```

Reply:
Actually this whole method is unused now, will remove.